flutter/flutter-provider/state-management-in-flutter-using-provider-Part2.md

```diff
@@ -0,0 +1,209 @@
+# State Management in Flutter using Provider: Part 2
+
+In previous [article](https://suresh.hashnode.dev/state-management-in-flutter-using-provider-part-1-cke8vk1cb00iny0s11lu933ch)
+we have discussed the problem of using `setState` to update the app's state in the moderate or complex app.
+
+## What we are going to know in this article
+
+1. Why we should use provider
+2. What are the types of provider
+3. How to use provider in our code
+
+In this article we'll know about `provider` package owned by  [Remi Rousselet](https://github.com/rrousselGit/provider), this is one of the best state management technique we can use in our app.
```

Comment:
comma after article  
can we use created instead of owned?

Reply:
Avoid adding opinion, saying "one of the most popular technique" is better than saying "best technique"

Reply:
change "we can use in our app." to "for flutter app development"

Reply:
Also, I think you're explaining what is Provider here, this should ideally go inside the first subheading "What is Provider and Why we use it".   
Don't use "In this article..." again, that's an opening statement. Jump straight to the main topic. Example:  
"Provider is one of the most popular state management solution created by ...."